app/controllers/tags_controller.rb

```diff
@@ -112,7 +116,7 @@ def feed
         raise ActiveRecord::RecordNotFound, "Couldn't find tag with id '#{params[:id]}'"
       end
       tag = tag.merger if !tag.canonical? && tag.merger
-      # Temp for testing
+      # F/F is currently a special case...
```

Comment:
Ok but why? If it's just because you need it to be canonical in testing then that is not a good solution. Why aren't you just making it canonical in the test?